### DISCUSSION TITLE
Error with mysql connector

### DISCUSSION THREAD

**Discussion by Shar-Tel** [pyinstaller/pyinstaller#7275] [Help] [answered]
Hi.
I have a python script that runs fine, but when I create a .exe with pyinstaller it fails when inserting data in a mysql database with the error:
Requested Module was not found: No module named 'mysql.connector.plugins'

I'm using Windows 10 and Python 3.11
Any idea of how to solve it?

**Comment by rokm** [comment, marked as answer]
Try adding `--hiddenimport mysql.connector.plugins` or perhaps `--collect-submodules mysql.connector` to your PyInstaller command.

**Comment by Shar-Tel** [comment]
--collect-submodules mysql.connector worked!!
Thanks a lot!
